tests/data/ar/Order.php

```diff
@@ -215,4 +216,14 @@ public function activeAttributes()
             0 => 'customer_id',
         ];
     }
+
+    public function getTrackingNumber($value)
```

Comment:
if we ever add something like this, the naming should not be `getX()` or `setX()` because such naming is reserved for the properties feature in Yii BaseObject.

Reply:
@cebe I'm going to rewrite a little bit this pull request, after some more discussion in linked issue.